lib/src/utils/config-reader/config_reader.dart

```diff
@@ -13,31 +14,39 @@ abstract class ConfigReader {
   bool? getBool(String key, {String? deprecatedKey});
   bool contains(String key);
 
-  /// By default this ConfigReader factory will try to load pubspec.yaml first.
-  /// If the sentry config doesn't exist on pubspec.yaml it will use sentry.properties as fallback.
+  /// This factory will try to load both pubspec.yaml and sentry.properties.
+  /// If a sentry config key doesn't exist on pubspec.yaml it will use sentry.properties as fallback.
```

Comment:
we should also document this somewhere, maybe sentry-docs?

Reply:
Added a task for myself. 👍